release-notes/3.1/3.1-known-issues.md

```diff
@@ -6,6 +6,25 @@ This document lists known issues for **.NET Core 3.1 Preview 1 and beyond releas
 
 Visual Studio 16.4 Preview 2 may uninstall .NET Core 3.0 when installed with the .NET Core 3.1 SDK. We recommend you re-install or repair the [.NET Core 3.0 SDK](https://dotnet.microsoft.com/download/dotnet-core/3.0) if this happens.
 
+### .NET Core 3.1.2 SDK (3.1.102 SDK)
+
+Packages built out of dotnet/corefx but part of the "Microsoft.AspNetCore.App" Shared Framework doesn't exist in the targeting pack 3.1.2. Due to this after upgrading to 3.1.2 apps which were able to compile against some assemblies can not longer compile. This will have an impact on both .NET Core as well as ASP.NET Core.
```

Comment:
```suggestion
Packages built out of dotnet/corefx but part of the "Microsoft.AspNetCore.App" Shared Framework don't exist in the targeting pack for 3.1.2. Due to this, after upgrading to 3.1.2, apps which were able to compile against some assemblies can not longer compile. This will have an impact on both .NET Core as well as ASP.NET Core.
```